modules/spark/src/main/scala/notebook/spark/util/SparkClosureUtil.scala

```diff
@@ -0,0 +1,59 @@
+package notebook.spark.util
+
+import scala.reflect.runtime.universe
+import org.apache.log4j.AppenderSkeleton
+import org.apache.log4j.spi.LoggingEvent
+
+class SparkClosureUtil(appendTo: String => Unit = println) {
+
+  // get the closure cleaner
+  private val runtimeMirror = universe.runtimeMirror(getClass.getClassLoader)
+  private val module = runtimeMirror.staticModule("org.apache.spark.util.ClosureCleaner")
```

Comment:
Yeah I know, but afterward the reflection will still be needed because their stuffed are marked private all over the places :-(.

But indeed I can at least remove some of them by first placing this in spark package